test/integration/features/basic.feature

```diff
@@ -64,6 +64,13 @@ Feature: Basic
       export "BAZ=BAT"
       """
 
+  Scenario: User is able to get url on running service command
+    Given Minishift has state "Running"
+     When executing "oc new-app https://github.com/openshift/nodejs-ex"
+      And executing "oc expose svc/nodejs-ex"
+      And executing "minishift openshift service nodejs-ex -n myproject --url"
+     Then stdout should match /nodejs-ex-myproject.(.*).nip.io/
```

Comment:
@gbraad , I don't think since, we need to be explicit here what output we are matching with. 

This can be output from `stdout` or `stderr`.